.distignore

```diff
@@ -0,0 +1,31 @@
+# Directories
+/.git
+/.github
+/.husky
+/.wordpress-org
+/assets
+/node_modules
+/vendor/bin
+
+# Files
+/.distignore
+/.editorconfig
+/.eslintignore
+/.eslintrc.json
+/.gitattributes
+/.gitignore
+/.lintstagedrc.json
+/.npmrc
+/.nvmrc
+/CHANGELOG.md
+/CODE_OF_CONDUCT.md
+/composer.json
+/composer.lock
+/CONTRIBUTING.md
+/CREDITS.md
+/LICENSE.md
+/package-lock.json
+/package.json
+/phpcs.xml
+/README.md
+/*.zip
```

Comment:
what's this here for?

Reply:
Added this in case plugin dirtro zip which was made by `npm run archive` script is present. It will never appear within the current push-deploy workflow, should we remove it?

Reply:
Will delete it for now, we can add it later when it's really required

Reply:
Yeah I don't think I've seen that creep in, so probably fine to remove but good idea to have ready in case that does end up happening here